nodeup/pkg/model/networking/amazon-vpc-routed-eni.go

```diff
@@ -35,8 +36,42 @@ func (b *AmazonVPCRoutedENIBuilder) Build(c *fi.NodeupModelBuilderContext) error
 		return nil
 	}
 
-	// Running Amazon VPC CNI on Ubuntu 22.04 and later requires setting MACAddressPolicy to `none` (ref: https://github.com/aws/amazon-vpc-cni-k8s/issues/2103 & https://github.com/kubernetes/kops/issues/16255)
-	if b.Distribution.IsUbuntu() && b.Distribution.Version() >= 22.04 {
+	if b.Distribution == distributions.DistributionAmazonLinux2023 {
+		// install iptables-nft in al2023 (NOT the iptables-legacy!)
+		c.AddTask(&nodetasks.Package{Name: "iptables-nft"})
+
+		// Mask udev triggers installed by amazon-ec2-net-utils package
+		c.AddTask(&nodetasks.File{
+			Path:     "/etc/udev/rules.d/99-vpc-policy-routes.rules",
+			Contents: fi.NewStringResource(""),
```

Comment:
just an empty file is enough here.

Reply:
I would add this as a code comment. My immediate thought here was that this must have been a mistake :)

Reply:
Done!